code/modules/projectiles/boxes_magazines/_box_magazine.dm

```diff
@@ -76,21 +76,28 @@
 	return 1
 
 /obj/item/ammo_box/attackby(obj/item/A, mob/user, params, silent = FALSE, replace_spent = 0)
-	if(INTERACTING_WITH(user, A))
+	if(INTERACTING_WITH(user, src) || INTERACTING_WITH(user, A))
```

Comment:
```suggestion
	if(INTERACTING_WITH(user, src))
```

Reply:
i tested it without both and half the time the check didn't work so